src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -14,9 +14,12 @@ def __init__(self, name, module):
         self.name = name
         self.type = type(module).__name__
 
-        self._forward = None
-
-
+def setattr_(model, name, module):
```

Comment:
This may not work if a high-level weighted module contains another low-level weighted.
I don't know if this situation is possible though.

Reply:
Could you give a example? this is used to replace a module by name in situation like `nn.sequential` where module name is like` xx.xx`.